lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -820,3 +833,41 @@ def test_certificate_custom_template_with_unicode_data(self, custom_certs_enable
                         else:
                             self.assertContains(response, "Tweet this Accomplishment")
                         self.assertContains(response, 'https://twitter.com/intent/tweet')
+
+    @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_certificate_branding(self):
```

Comment:
We should be testing for the negative cases/branches here as well -- ie, what happens if a URL is not defined, or if it returns EMPTY_URL, etc.